src/popup/router/components/MenuCarousel.vue

```diff
@@ -34,30 +50,35 @@ import Invites from '../../../icons/invites.svg?vue-component';
 export default {
   name: 'MenuCarousel',
   components: {
-    Flickity,
     BoxButton,
+    Arrow,
     Balances,
     Payments,
     Tips,
     Activity,
     Names,
     Invites,
   },
-  data() {
-    return {
-      flickityOptions: {
-        initialIndex: 0,
-        prevNextButtons: true,
-        pageDots: false,
-        freeScroll: false,
-        draggable: process.env.PLATFORM === 'cordova',
```

Comment:
flickity was swipeable on cordova, but the current approach is not, consider implementing it according to #894